release/models/platform/openconfig-platform-controller-card.yang

```diff
@@ -73,6 +79,22 @@ module openconfig-platform-controller-card {
     uses controller-card-config;
   }
 
+  augment "/oc-platform:components/oc-platform:component/" +
+    "oc-platform:controller-card/oc-platform:config" {
+        description
+          "Configuration data for controller card components.
+          A controller-card can be configured for persistent
+          powered-off mode using the config/power-admin-state leaf.
+          The reference path below defines rules for such a
+          configuration.";
+
+        reference
+          "Rules around power-off configuration in controller-cards:
+          https://github.com/openconfig/public/blob/master/doc/controller-card_poweroff.md";
```

Comment:
This is a duplicate augment.  To fix it you can remove this augment clause and move the description and reference to replace lines 66-68

Reply:
Done